learning/step_by_step/instancing.rst

```diff
@@ -91,22 +91,21 @@ Cool, huh? This is how instancing works.
 Editing instances
 -----------------
 
-Select one of the many copies of the balls and go to the property editor. Let's
-make it much more bouncy. To do this, look for the Bounce parameter and set it
-to 1.0:
+Select one of the many copies of the balls and go to the property
+editor. Let's make it bounce a lot more, so look for the Bounce
+parameter and set it to 1.0:
 
 .. image:: /img/instedit.png
 
-Notice that a grey "revert" button appeared. When this button is present, it
-means we modified a property in the instanced scene to override a specific
-value that differs from the original scene. Even if that property is modified
-in the original scene later, our custom value here will continue to overwrite
-it. Pressing the revert button will restore the property to the value that
-comes from the original scene.
+Grey "revert" button will appear. When
```

Comment:
Not a fragment, but might use an article before "gray", e.g. *__A__ gray "revert" button will appear*